db/model.js

```diff
@@ -19,4 +18,21 @@ exports.selectAllEndPoints = () => {
     .then((data) => {
         return JSON.parse(data);
     });
-    };
\ No newline at end of file
+    };
+
+exports.selectArticles = (article_id) => {
+    if (!Number(article_id)) {
```

Comment:
So psql will handle this for you. This is currently being handled by your custom error handler and your psql handler isn't firing. In theory if you remove this if condition your testing should still pass as you have the correct error handler in place.